src/jquery.autofilter.js

```diff
@@ -80,6 +88,27 @@
             $("body").append(modalWidgetCode);
 
             modalWidget = $("#" + modalWidgetId);
+            $inputText = $('input.filter-input')
```

Comment:
Can you add semi-colons to all the lines to keep it consistent?

Reply:
oops, fixing it.